mappings/net/minecraft/item/SpawnEggItem.mapping

```diff
@@ -1 +1,4 @@
 CLASS ayt net/minecraft/item/SpawnEggItem
+	FIELD a entityTypesToSpawnEggs Ljava/util/Map;
```

Comment:
```
private static final Map<EntityType<?>, SpawnEggItem> field_8914 
```

I see a constant, but not in the name...

Reply:
This can just be `SPAWN_EGGS`